java/lib/bcel.astub

```diff
@@ -0,0 +1,8 @@
+import org.checkerframework.checker.index.qual.SameLen;
```

Comment:
This stub file should appear in the Checker Framework, not in plume-lib

Reply:
That makes sense. I'll do that.

Reply:
See https://github.com/typetools/commons-bcel/pull/4